app/components/flash-messages.js

```diff
@@ -5,8 +5,27 @@ const {
   inject: { service }
 } = Ember;
 
+/**
+  `flash-messages` is a container for the apps flash messages. It displays the
```

Comment:
Minor note, but just an apostrophe missing in "app's".

I might simplify though to:

> `flash-messages` is a container for displaying flash messages.